eureka-client/src/main/java/com/netflix/discovery/converters/EurekaJacksonCodec.java

```diff
@@ -230,7 +230,7 @@ public Name getName() {
                 };
             }
 
-            Map<String, String> metaData = new HashMap<>();
+            Map<String, String> metaData = new HashMap<String, String>();
```

Comment:
We are at Java7 language level, so adding type parameters here on the left side is redundant.